### PR TITLE
Docs update 2017-08-14

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

File: 3-0-libraries.md [hunk 1]
```diff
@@ -165,36 +191,37 @@ made previously.
...
 
 Pools are the last step for libraries before sequencing and represent the
 entity that is loaded onto the flowcell lane or SMRTcell. A "pool" can have one or more
-library dilutions in it. They are equivalent to "worksets" in Geospiza, but are
-required for every lane of sequencing.
+library dilutions in it. Every lane of sequencing contains only one pool. 
 
 <img src="pics/flow-pool.svg"/>
 
 Here we will all all of the dilutions we added previously to make a single pool
```
Comment (djcooke) [inline 3-0-libraries.md]: Could you fix the typo here while you're at it? "Here we will **pool** ~~all~~ all of the dilutions..."

File: 3-0-libraries.md [hunk 2]
```diff
@@ -165,36 +191,37 @@ made previously.
...
-of 4.
+of 4 samples.
```
Comment (djcooke) [inline 3-0-libraries.md]: MISO "samples" don't get pooled, so they should be specifically referred to as dilutions in documentation at this point

File: 3-0-libraries.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Download the worksheet for this section here: <a href="3-0-libraries-worksheet">
 A _library_ is made from one sample for a single target _platform_ and
 has a specific _design_ associated with it that decides the _selection_
 and _strategies_ used to make the library. A library may also have _indices_
-(primers) and QC information.
+(primers/barcodes/molecular IDs) and QC information.
```
Comment (callunity) [inline 3-0-libraries.md]: I think it's worth keeping something like "barcodes" in for training new users, who may be coming from somewhere where they call them barcodes. Perhaps "sequencing barcodes"?

Comment (djcooke) [inline 3-0-libraries.md]: Sure, less ambiguous :+1:

**Pull request merged**